build/Microsoft.DotNet.Cli.Monikers.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <SdkBrandName>.NET Core SDK 1.1.10</SdkBrandName>
+    <SdkBrandName>.NET Core SDK 1.1.11</SdkBrandName>
     <SharedFrameworkBrandName>Microsoft .NET Core 1.1.9 - Runtime</SharedFrameworkBrandName>
```

Comment:
Is the runtime version bumping too?

Reply:
It is, but we will update that separately.